### ISSUE TITLE
Vertex attribute at location x not consumed by vertex shader

### ISSUE THREAD

**Issue by MilanDierick** [Overv/VulkanTutorial#315]
TL;DR I've compiled the source code provided by https://vulkan-tutorial.com/Vertex_buffers/Vertex_buffer_creation and it's the exact same issue, which makes me believe my implementation is correct, but there is some other problem. Did something change in the past years with Vulkan that would invalidate this tutorial? From what I've read a lot of people use this website to get into Vulkan, surely I'm not the only one experiencing this issue?

Refer to my Stack Overflow question for all the details: https://stackoverflow.com/questions/74817370/vulkan-vertex-attribute-not-consumed-by-vertex-shader

**Comment by MilanDierick** [comment]
Resolved, my cmake script responsible for recompiling the shaders was incorrect 🤦
